### PR TITLE
Feat/mobility

### PR DESCRIPTION
**Pull request by sourabhsaini11**

### CODE REVIEW

File: constants/index.ts [hunk 1]
```diff
@@ -24,6 +24,36 @@ export default Object.freeze({
   DECIMAL_PRECISION: 6,
   ORDER_PICKED: 'Order-picked-up',
   ORDER_DELIVERED: 'Order-delivered',
+  FIS_SEARCH: 'search',
```
Comment (bluecypher) [inline constants/index.ts:27]: Pls merge the constants with same values. eg RET_SEARCH, FIS_SEARCH, MOB_SEARCH and replace it across all the files

File: controller/validate/index.ts [hunk 2]
```diff
@@ -28,6 +30,32 @@ const controller = {
...
+  validateFIS: async (req: Request, res: Response): Promise<void> => {
+    try {
+      const { domain, version, payload, flow } = req.body
+      let response
+
+      if (!(flow in fisFlows)) {
+        res.status(400).send({ success: false, response: 'Invalid Flow!!' })
+      } else {
+        switch (version) {
+          case '1.2.0':
```
Comment (bluecypher) [inline controller/validate/index.ts:42]: This switch case is for retail version 1.2.0. pls utilize the validate function itself and accommodate all the domains there

**Review by bluecypher — CHANGES_REQUESTED** [review]
Can you please fix the issues

**Comment by sourabhsaini11** [comment]
@bluecypher above comments are resolved, kindly review

**Review by bluecypher — CHANGES_REQUESTED** [review]
package-lock.json and .DS_Store files should be a part of gitignore and not to be uploaded on GitHub. Please check the feat/phase2 branch for such minor fixes

**Pull request merged**